sandbox/watch.sh

```diff
@@ -19,6 +19,6 @@ tsc --build -w --preserveWatchOutput $PROJECT &
 css_files="app/client/**/*.css"
 chokidar "${css_files}" -c "bash -O globstar -c 'cat ${css_files} > static/bundle.css'" &
 webpack --config $WEBPACK_CONFIG --mode development --watch &
-NODE_PATH=_build:_build/stubs:_build/ext nodemon ${NODE_INSPECT:+--inspect} --delay 1 -w _build/app/server -w _build/app/common _build/stubs/app/server/server.js &
+NODE_PATH=_build:_build/stubs:_build/ext nodemon ${NODE_INSPECT} --delay 1 -w _build/app/server -w _build/app/common _build/stubs/app/server/server.js &
```

Comment:
What does this change do? :)

Reply:
It's unrelated, but I explain it in the best place to hide sensitive secrets: [in the commit message](https://github.com/gristlabs/grist-core/pull/973/commits/f0c11c5d98a24b7ca08659d545735a8751dc823a).

In case you didn't know (and sorry if you did), `--inspect` will start and run a Node process that you can attach a debugger, whereas `--inspect-brk` starts the process but immediately sets a breakpoint at the entry point of execution. The latter is helpful if you want to start debugging a process from the very beginning. In the case of Grist, I wanted to debug initialisation to understand how the server starts listening for websocket connexions.

Reply:
To add something about that: using `yarn start:debug` is not perfect as it requires to close the debugger for letting nodemon restart the node process… That's probably a nodemon / node limitation.

Unless there's something new since the last time I used this command.